.github/CODEOWNERS

```diff
@@ -925,7 +927,10 @@ lib/periodic_jobs.rb @department-of-veterans-affairs/va-api-engineers @departmen
 lib/preneeds @department-of-veterans-affairs/mbs-core-team @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/rx @department-of-veterans-affairs/mobile-api-team @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
 lib/saml @department-of-veterans-affairs/vsp-identity
+lib/salesforce @department-of-veterans-affairs/backend-review-group
```

Comment:
`lib/salesforce` was added in the same PR (https://github.com/department-of-veterans-affairs/vets-api/pull/2184) as `lib/gibft` (i think it stands for GI Bill Feedback Tool) so you'll also want this to be owned by @department-of-veterans-affairs/my-education-benefits

Reply:
Same as above, I would recommend `@department-of-veterans-affairs/govcio-vfep-codereviewers`